packages/core/src/2d/text/Font.ts

```diff
@@ -14,9 +15,26 @@ export class Font extends RefObject {
    * Create a font from OS.
    * @param engine - Engine to which the font belongs
    * @param name - The name of font
+   * @param fontUrl - The font url to register, if not, will use system font
    * @returns The font object has been create
    */
-  static createFromOS(engine: Engine, name: string = ""): Font {
+  static async create(engine: Engine, name: string, fontUrl: string = ""): Promise<Font> {
```

Comment:
Use function overloading

Reply:
Just one api for user, create, createFromOS should be internal